cmd/cmd.go

```diff
@@ -24,34 +24,26 @@ var (
 	disallowAdditionalProperties bool
 	overwrite                    bool
 	allowEmpty                   bool
+	requireAll                   bool
 	outputStdOut                 bool
 	output                       string
 	input                        string
+
+	errReturned error
 )
 
 // rootCmd is the base command for terraschema
 var rootCmd = &cobra.Command{
-	Use:   "terraschema",
-	Short: "Generate JSON schema from HCL Variable Blocks in a Terraform/OpenTofu module",
-	Long:  `TODO: Long description`,
-	Run: func(cmd *cobra.Command, args []string) {
-		path, err := filepath.Abs(input) // absolute path
-		if err != nil {
-			fmt.Printf("could not get absolute path: %v\n", err)
-			os.Exit(1)
-		}
-		output, err := jsonschema.CreateSchema(path, false)
-		if err != nil {
-			fmt.Printf("error creating schema: %v\n", err)
-			os.Exit(1)
-		}
-		jsonOutput, err := json.MarshalIndent(output, "", "    ")
-		if err != nil {
-			fmt.Printf("error marshalling schema: %v\n", err)
+	Use:     "terraschema",
+	Example: "terraschema -i /path/to/module -o /path/to/schema.json",
+	Short:   "Generate JSON schema from HCL Variable Blocks in a Terraform/OpenTofu module",
+	Long:    `TODO: Long description`,
```

Comment:
Is it worth filling in the long description as part of this pr?

Reply:
I was waiting until I'd acutally written the documentation but I can add it here now that that exists.